src/components/editor/FileSelector.tsx

```diff
@@ -51,7 +51,7 @@ const columns: GridColDef[] = [
 ];
 
 function EditorFileSelector() {
-    const [initDone, setInitDone] = useState(false);
+    const initDone = useRef(false);
```

Comment:
Using a ref makes more sense here

Reply:
I'll just trust you on that one :upside_down_face: